src/ontology/mondo-edit.obo

```diff
@@ -565000,6 +564979,27 @@ synonym: "variable age onset epilepsy" BROAD [https://orcid.org/0000-0001-5208-3
 is_a: MONDO:0015650 {source="https://www.epilepsydiagnosis.org/syndrome/ffevf-overview.html#"} ! epilepsy syndrome
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8455" xsd:string
 
+[Term]
+id: MONDO:0100620
+name: developmental and epileptic encephalopathy
+def: "An epilepsy associated with developmental impairment that may be due to either the underlying etiology or the superimposed epileptic activity, or both." [PMID:35503715]
```

Comment:
I don't understand the definition. "due to either the underlying etiology or superimposed epileptic activity"

Reply:
Ingo said this is the definition we should use. I don't think we should change it.

Reply:
Even though Ingo agreed, it doesn't make sense to me. what does "the underlying etiology" means? is this referring to the developmental impairment part that can be due to the underlying etiology of the epilepsy? This needs to be expressed more clearly.